### ISSUE TITLE
Project translation?

### ISSUE THREAD

**Issue by Thomasfda** [filfat/swiftsnapper#8]
Good evening , I am French , and I would like to know if the project will be translated later :) ? (Sorry my english is bad)

**Comment by gncgabriel** [comment]
@Thomasfda hello, I am Brazilian and my English is too bad,  I myself already started translating some things in the app

**Comment by Thomasfda** [comment]
Hi @gncgabriel i think started translation at french

**Comment by gncgabriel** [comment]
would be cool something that when the user opens the APP he could choose the language of the app, but I think too early for that

**Comment by Thomasfda** [comment]
I also think I'll see for a translation and I'll post a job if the translation is accepting :) !

**Comment by allan991** [comment]
He need think about it but it's take more time to translates to foreign languages

**Comment by Thomasfda** [comment]
I think I can facilitate its work

**Comment by utopy** [comment]
I'm italian and good with English... I can help you with italian translation!

**Comment by Thomasfda** [comment]
![regis](https://cloud.githubusercontent.com/assets/11091332/12222996/b5c8f83c-b7cb-11e5-9f69-6f6cf72e978e.jpg)
![co](https://cloud.githubusercontent.com/assets/11091332/12222997/b5ca40ac-b7cb-11e5-9f49-3aac98006d82.jpg)

I have started a translation :)

**Comment by Draco351** [comment]
I could help with spanish translation if you want ;)

**Comment by pvrs12** [comment]
How did you go about translating this? Did you just edit the hard-coded strings in the project? 

I'm going to probably add a strings localization file so that translations can be easier to plug in and out based on language

**Comment by gncgabriel** [comment]
![wp_ss_20160110_0009 1](https://cloud.githubusercontent.com/assets/16600779/12223133/3c4557b8-b7b6-11e5-863c-754ef9962782.png)
![wp_ss_20160110_0007 1](https://cloud.githubusercontent.com/assets/16600779/12223134/404129c8-b7b6-11e5-9adb-3d30733c021a.png)

:)

**Comment by Thomasfda** [comment]
I translate on files .html
I will create another subject for French translation

**Comment by filfat** [comment]
Hi, I'll quickly make a commit including json files with the english strings :)

**Comment by Thomasfda** [comment]
Cool i translate the file after :D !

**Comment by hayaleturk** [comment]
Hi, can I help for Turkish language?

**Comment by kikothiago** [comment]
Good to see Brazil working together on this project. Congratulations to all of you.

**Comment by MuriloPereira2015** [comment]
Hi! I can help translate the app to PT-BR too.

**Comment by YBTD** [comment]
I can help with translation to dutch if you want

**Comment by Maxyeah** [comment]
I can help translate into german :)

**Comment by Dafinek** [comment]
me with Czech :)

**Comment by christvedt** [comment]
I would love to help with Norwegian. How can I help?

**Comment by YBTD** [comment]
I will also provide updates to the dutch languages files, but I think now the focus is on building up the app. I will keep following and translate new added strings later

**Comment by vguille** [comment]
I want to help translate for pt-BR

**Comment by allan991** [comment]
Well I see how many people here want's to help to build the app and translate, however its not so simple but I can help you guys showing some videos about the apache Cordova

**Comment by a4x2** [comment]
I wanna help with translation to turkish if you want to

**Comment by eduardoribeiro** [comment]
I'd love to help with the portuguese Portugal (pt-PT) translation

**Comment by sefor** [comment]
![wp_ss_20160113_0001](https://cloud.githubusercontent.com/assets/16676159/12281504/9c965b7e-b98e-11e5-907f-24130673ab08.png)
![wp_ss_20160113_0001](https://cloud.githubusercontent.com/assets/16676159/12281509/aa42ba88-b98e-11e5-8481-eddd12acda37.png)

@eduardoribeiro , already done by someone.

colors not the same comparing to the previous posts

**Comment by filfat** [comment]
@sefor I've changed them to Cyan which is a lot easier for the eyes and helps creating our own identity :)

**Comment by allan991** [comment]
@filfat I agree. We need create your own identity for the app and change the yellow to cyan its ok however some elements of snapchat (the best of)  app need be in something like a: A ghost or something very similar to final user thinks (this is a client but its already Snapchat)

**Comment by realfakenerd** [comment]
The translation to PT-BR seems to be ok

**Comment by sefor** [comment]
@fifat ok, I understand and agree 😊

**Comment by JJohnson1701** [comment]
I'd be happy to help translate into German, and other languages (Icelandic, Old Norse, Old English, Gothic, Saxon/Low German, Frisian, etc).

**Comment by allan991** [comment]
We need now volunteers to translate the app in Chinese Mandarim and Japanese

**Comment by yemreyazici** [comment]
Hello. I am from turkey. I can translate to turkish

**Comment by marcus2310** [comment]
I can help you traslating to spanish and catalán

**Comment by mentos1386** [comment]
@marcus2310 [here](https://github.com/filfat/swiftsnapper/tree/master/www/lang) are translation files, look around to see if there are mistakes or add some language that isn't added yet. But be careful, you have to maintain the same format/style of code. [here](https://github.com/filfat/swiftsnapper/blob/master/www/lang/es-ES.json) is a Spanish translation file.

**Comment by synialke691** [comment]
I can translate into Polish
